### DISCUSSION TITLE
Change Web Server Cert - Docker

### DISCUSSION THREAD

**Discussion by skylersll** [Keyfactor/ejbca-ce#674] [General]
I have installed ejbca-ce via docker. I can not figure out how to update the web server cert from the default management CA to either my new root CA or to update the dns name of the existing. I do not see this anywhere in the documentation, any ideas?

**Comment by primetomas** [comment]
See here, towards the bottom. https://hub.docker.com/r/keyfactor/ejbca-ce
Look at all the bottom for all environment variables. There is an example in the end how to use your own keystore files.
